data/coalition/coalition missions.txt

```diff
@@ -1684,3 +1684,180 @@ mission "Coalition: Alpha Encounter"
+			`	"A scientist, I am, and invitied to speak at a university seminar, I have been." This time, he speaks in a calmer tone of voice. "Be there by <date>, I must. The honor of transporting us to Shadow of Leaves, will you give me?"`
+			choice
+				`	"What is the seminar about?"`
+					goto "xeno"
+				`	"Sure, I'd be glad to transport you."`
+					goto "accept"
+				`	"Sorry, I can't take transport jobs right now."`
+					decline
+			label "xeno"
+			`	"Funny you ask, it is. Xenologist, I am. Alien culture and physiology, I study, and the topic of the seminar, it is. Invited to give lectures on the subject, I have been. Take me there, will you?"`
+			choice
+				`	"Sure, I'd be glad to transport you."`
+				`	"Sorry, I can't take transport jobs right now."`
+					decline
+			label "accept"
+			`	"Wonderful!" he says. "Aelbris, I am." He soon gathers his things and boards your ship. As he enters, he seems fascinated by its structure and design. His gawking makes it difficult to get him to finally settle down into his bunk.`
+					accept
+	on complete
+		"coalition jobs" += 3
+		payment 232000
+		conversation
+			`During the trip, Aelbris was polite, if a little excited to have a human as his pilot. He asked you several questions during the flight, which you answered as best as you can. Once you and your passenger departs from your ship, Aelbris turns around.`
+			`	"Captain, that you must be very busy, I realize, but wondering if you could stay for the seminar, I was. You see, studying human space for years, I have been, but actually meet a human, I have not. A great boon to our field to have a human speak at our seminar, it would be."`
+			choice
+				`	"What do I have to do there?"`
+					goto "answer"
+				`	"I don't think that I'd be a good representative of humanity."`
+					goto "low"
+				`	"How do you manage to study humanity without meeting any humans?"`
+					goto "explanation"
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
+			label "answer"
+			`	"Answering some questions from the attendees, would you be comfortable with? Simple, they would be, about aspects of human society, physical structure, and the like, they would pertain to."`
+			choice
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
+			label "low"
+			`	"Nonsense! Travel the galaxy, you do. A most knowledgable individual, surely you must be. Accept my offer, will you?"`
+			choice
+				`	"Sure, I'd be glad to."`
+					goto "yes"
+				`	"Sorry, I don't have time."`
+						decline
+			label "explanation"
+			`	"Close to Arachi space, humans are, so study radio and TV signals from your part of the galaxy, we are able to. Faint, scattered broadcasts, sophisticated receivers collect, and the signals to us, Arachi colleagues relay. From these broadcasts, we learn. Help us learn more, would you?"`
```

Comment:
This does raise a question - if the Coalition can study humans in this way, how has humanity not noticed similar signals from the Coalition?

Reply:
According to this https://web.archive.org/web/20160708171434/http://setifaq.org/faq.html#1.2.3, humanity, using current technology, would not be able to detect broadband even a fraction of a light year away from Earth. We can assume that technology in the year 3014 has improved somewhat, but I think that it's plausible that we are still unable to detect broadcasts from the Coalition, whose planets are several light years away from us. The Coalition, because it's been around a lot longer, probably has more sophisticated technology that is able to detect human broadcasts. If anyone more knowledgeable about this can tell me if this reasoning is bull, please do so.

Reply:
Fair enough. I don't know a whole lot about the subject, so that seems like a reasonable explanation to me.